apps/dotcom/sync-worker/src/TLUserDurableObject.ts

```diff
@@ -272,8 +273,18 @@ export class TLUserDurableObject extends DurableObject<Environment> {
 						`Cannot create a file for another user ${nextFile.id}`
 					)
 				}
+				// Owners are allowed to make changes
 				if (prevFile.ownerId === this.userId) return
-				if (prevFile.shared && prevFile.sharedLinkType === 'edit') return
+
+				// We can make changes to updatedAt field in a shared, editable file
+				if (prevFile.shared && prevFile.sharedLinkType === 'edit') {
```

Comment:
We used to just allow everything if file was shared in edit mode (even for non owners). We know only allow changing `updateAt` field.